docs/tutorials/using_dpdk.md

```diff
@@ -0,0 +1,618 @@
+|--------------|---------------|------------|
+| `management` | 10.201.0.0/24 | 10.201.0.1 |
+| `access`     | 10.202.0.0/24 | 10.202.0.1 |
+| `core`       | 10.203.0.0/24 | 10.203.0.1 |
+| `ran`        | 10.204.0.0/24 | 10.204.0.1 |
+
+On the host machine, create local network bridges to be used by LXD by adding below configuration under `/etc/netplan/99-sdcore-networks.yaml`.
+Before creating the configuration of the network bridges, please make sure that:
+
+- mgmt-br route metric value is higher than your default route's metric
+- core-br metric value is higher than your mgmt-br route's metric
+
+Change the metrics of SD-Core routes which are indicated with comments below, relatively to your default route's metric if required.
+
+```shell
+cat << EOF | sudo tee /etc/netplan/99-sdcore-networks.yaml
+# /etc/netplan/99-sdcore-networks.yaml
+network:
+  bridges:
+    mgmt-br:
+      addresses:
+        - 10.201.0.14/24
+      routes:
+        - to: default
+          via: 10.201.0.1
+          metric: 110 # Set the value higher than your default route's metric
+    access-br:
+      addresses:
+        - 10.202.0.14/24
+      routes:
+        - to: 10.204.0.0/24
+          via: 10.202.0.1
+    core-br:
+      addresses:
+        - 10.203.0.14/24
+      routes:
+        - to: default
+          via: 10.203.0.1
+          metric: 203 # Set the value higher than your mgmt-br route's metric
+    ran-br:
+      addresses:
+        - 10.204.0.14/24
+      routes:
+        - to: 10.202.0.0/24
+          via: 10.204.0.1
+  version: 2
+EOF
+```
+
+Arrange the file permissions and apply the network configuration:
+
+```shell
+sudo chmod 600 /etc/netplan/99-sdcore-networks.yaml
+sudo netplan apply
+```
+
+```{note}
+Applying new netplan configuration may produce warnings related to file permissions being too open. 
+You may safely disregard them.
+```
+
+```{note}
+`ran-br` bridge is only used to create a route from `access` to `ran` network in User Plane Function.
+```
+
+### Install and Configure LXD
+
+Install LXD:
+
+```console
+sudo snap install lxd
+```
+
+Initialize LXD:
+
+```console
+lxd init --auto
+```
+
+### Install and configure Multipass
+
+Install Multipass:
+
+```console
+sudo snap install multipass
+```
+
+Set LXD as local driver:
+
+```console
+multipass set local.driver=lxd
+```
+
+Connect Multipass to LXD:
+
+```console
+sudo snap connect multipass:lxd lxd
+```
+
+## 2. Create a Virtual Machine
+
+To complete this tutorial, you will need one virtual machine with access to the networks as follows:
+
+| Machine                              | CPUs | RAM | Disk | Networks                       |
+|--------------------------------------|------|-----|------|--------------------------------|
+| User Plane Kubernetes Cluster        | 4    | 12g | 20g  | `management`, `access`, `core` |
+
+Create VM with Multipass:
+
+```console
+multipass launch -c 4 -m 12G -d 20G -n user-plane  --network mgmt-br --network core-br --network access-br jammy
```

Comment:
We reserved the huge pages on the host, but we are not launching the VM using those huge pages. Looking at the multipass documentation, I cannot see a method of launching the VM to use that huge page. If MP cannot, then we should remove the huge page configuration from the host machine configuration as it is locking the 2G RAM that nothing can use.

Reply:
I just did a check on my laptop. No huge pages configured
```
mark@mark-laptop:~$ grep Huge /proc/meminfo
AnonHugePages:         0 kB
ShmemHugePages:   831488 kB
FileHugePages:         0 kB
HugePages_Total:       0
HugePages_Free:        0
HugePages_Rsvd:        0
HugePages_Surp:        0
Hugepagesize:       2048 kB
Hugetlb:               0 kB
```

Launched a MP vm, added 1g huge pages to grub, and it has huge pages after reboot
```
ubuntu@user-plane:~$ grep Huge /proc/meminfo 
AnonHugePages:         0 kB
ShmemHugePages:        0 kB
FileHugePages:         0 kB
HugePages_Total:       2
HugePages_Free:        2
HugePages_Rsvd:        0
HugePages_Surp:        0
Hugepagesize:    1048576 kB
Hugetlb:         2097152 kB
```

So it is very unlikely that multipass is actually using the host.

Reply:
I tested it again and you are right. Enabling hugepages in the host part is unnecessary. This part is removed.